client/state/sites/selectors.js

```diff
@@ -148,29 +147,22 @@ export function isJetpackSite( state, siteId ) {
 }
 
 /**
- * Returns true if site is a Pressable Jetpack site or site during a transfer, false if the site is hosted on
- * WordPress.com or is a regular Jetpack, or null if the site is unknown.
+ * Returns true if site is a Automated Transfer site, false if not and null if unknown
```

Comment:
Should we make a distinction between whether we know if the site is not an AT site and if we don't have this info? I think it's enough to return `true` if we have the info and the site is AT and `false` otherwise.